sql/reference/01-buildings_reference_schema.sql

```diff
@@ -0,0 +1,129 @@
+------------------------------------------------------------------------------
+-- Create buildings reference schema and tables
+------------------------------------------------------------------------------
+
+SET client_min_messages TO WARNING;
+
+
+CREATE SCHEMA IF NOT EXISTS buildings_reference;
+
+-- Suburb / Locality
+
+CREATE TABLE IF NOT EXISTS buildings_reference.suburb_locality (
+      suburb_locality_id serial PRIMARY KEY
+    , external_suburb_locality_id integer
+    , suburb_4th character varying(60)
+    , suburb_3rd character varying(60)
+    , suburb_2nd character varying(60)
+    , suburb_1st character varying(60)
+    , shape public.geometry(MultiPolygon, 2193)
+);
+
+-- Town / City
+
+CREATE TABLE IF NOT EXISTS buildings_reference.town_city (
+      town_city_id serial PRIMARY KEY
+    , external_city_id integer
+    , name character varying(60)
+    , shape public.geometry(MultiPolygon, 2193)
+);
+
+-- Territorial Authority
+
+CREATE TABLE IF NOT EXISTS buildings_reference.territorial_authority (
+      territorial_authority_id serial PRIMARY KEY
+    , external_territorial_authority_id integer
+    , name character varying(100)
+    , shape public.geometry(MultiPolygon, 2193)
+);
+
+-- Coastline
+
+CREATE TABLE IF NOT EXISTS buildings_reference.coastlines (
+      coastline_id serial PRIMARY KEY
+    , shape public.geometry(MultiPolygon, 2193)
+);
+DROP INDEX IF EXISTS shx_coastlines;
+CREATE INDEX shx_coastlines
+    ON buildings_reference.coastlines USING gist (shape);
+
+-- River Centrelines
+
+CREATE TABLE IF NOT EXISTS buildings_reference.river_centrelines (
+      river_centreline_id serial PRIMARY KEY
+    , shape public.geometry(MultiLinestring, 2193)
```

Comment:
River centrelines are just `Linestring`s. I believe the `Multi` can be removed from all except the Coastlines and Islands geometry.

Reply:
Test data will need to be updated as well to suit. You can just wrap the existing geometry in a `(ST_Dump(<shape>)).geom` to get the non-multi geometry out ..